Casks/subethaedit.rb

```diff
@@ -0,0 +1,11 @@
+cask 'subethaedit' do
+  version '5.0'
+  sha256 '22c4219873a73602794805ea26d671e914dcf667041ac7508beb2ad5b6cda982'
+
+  url 'https://subethaedit.net/Releases/SubEthaEdit-5.0rc1.zip'
```

Comment:
You can use string interpolation here to reuse the version.

In this case, if you choose to stick with the `5.0rc1` as a `version` stanza value:

```suggestion
  url "https://subethaedit.net/Releases/SubEthaEdit-#{version}.zip"
```

Reply:
updated the feed and binary so this can work now. (e.g. put SubEthaEdit-5.0.zip in the appcast, and made the binary accessible as such too. The app never showed the rc1 anyways, so it is a pre-release artifact that it was in there)

Reply:
Thanks for the hint. I updated the PR with `version` in url.